### DISCUSSION TITLE
Error 404 (I'm kind of a noob on git)

### DISCUSSION THREAD

**Discussion by ACKRONNE** [community/community#37923] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! everyone I'm trying to upload my web page on git but.. I have a 404 problem https://ackronne.github.io/Okoticket/ that is the web page. 
I already check the repository files and works normally on xampp but not in here. What else can I check on?

**Comment by airtower-luna** [comment, marked as answer]
At a glance, your repository seems to contain a Java servlet. That won't work on Pages. GitHub Pages allows only static sites, nothing that runs on the server. What you *can* do is run a static site generator in Actions and deploy the resulting site to Pages, but that has very little to do with servlets.

**Reply by ACKRONNE** [reply]
Oh! I see, Thank you so much
